receiver/rabbitmqreceiver/integration_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration
+// +build integration
+
+package rabbitmqreceiver
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+)
+
+var (
+	containerRequest3_9 = testcontainers.ContainerRequest{
+		FromDockerfile: testcontainers.FromDockerfile{
+			Context:    filepath.Join("testdata", "integration"),
+			Dockerfile: "Dockerfile.rabbitmq.3_9",
+		},
+		ExposedPorts: []string{"15672:15672"},
+		Hostname:     "localhost",
+		WaitingFor: wait.ForListeningPort("15672").
+			WithStartupTimeout(2 * time.Minute),
+	}
+)
+
+func TestRabbitmqIntegration(t *testing.T) {
+	t.Run("Running rabbitmq 3.9", func(t *testing.T) {
+		t.Parallel()
+		container := getContainer(t, containerRequest3_9)
+		defer func() {
+			require.NoError(t, container.Terminate(context.Background()))
+		}()
+		hostname, err := container.Host(context.Background())
+		require.NoError(t, err)
+
+		f := NewFactory()
+		cfg := f.CreateDefaultConfig().(*Config)
+		cfg.Endpoint = fmt.Sprintf("http://%s:15672", hostname)
+		cfg.Username = "otel" //otel guest
+		cfg.Password = "otel"
```

Comment:
Don't these need to be updated as well? I wonder if we'll see more metrics if these are updated.

Reply:
ok